frontend/utilities/generate_csv/index.ts

```diff
@@ -18,13 +18,14 @@ export const generateCSVFilename = (descriptor: string) => {
 export const generateCSVQueryResults = (
   rows: Row[],
   filename: string,
-  tableHeaders: Column[] | string[]
+  tableHeaders: Column[] | string[],
+  omitHostDisplayName?: boolean
```

Comment:
Not a blocker, but when function params are more than 3, I like to think about refactoring into a params object:

```ts
generateCSVQueryResults = ({ rows, filename, tableHeaders, omitHostDisplayName = false}: {
  rows: Row[];
  filename: string;
  tableHeaders: Column[] | string[];
  omitHostDisplayName?: boolean
}) => {
 ...
}
```

Reply:
nice I like that as a guiding principle